### ISSUE TITLE
Aiding Developers in Understanding the Specification Through AI

### ISSUE THREAD

**Issue by codehag** [tc39/tg5#31]
This isn't something we are actively doing, but something that MDN is currently experimenting with. We can use this as a tracking issue, and also as a place to gather related literature. (seems there have been some failures in this space already, lets find them)

Discussed in https://github.com/tc39/tg5/blob/main/meetings/2024-05-29.md

**Comment by asukaminato0721** [comment]
I thought <https://notebooklm.google.com/> could be a good example, since it can answer the question based on the material, and give position from the material.

**Comment by ecmageneva** [comment]
Yes, I agree it is a very good example. Worth to play with it. But at the moment to me it is definitely not a "final stage" tool yet. But it will.
